caddy/caddy.go

```diff
@@ -323,9 +323,9 @@ func (f *FrankenPHPModule) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 						if d.NextArg() {
 							return d.ArgErr()
 						}
+					} else {
+					    return d.ArgErr()
```

Comment:
Actually, we should do something like that to return the original error:

```golang
					v, err := strconv.ParseBool(d.Val());
					if  err != nil {
						return err
					}
					if d.NextArg() {
						return d.ArgErr()
					}

					f.ResolveRootSymlink = &v
```

(quick note: we use tabs for indentation, not spaces)

Reply:
Could you keep this block please, it will return the exact reason why the value cannot be parsed to the user.

Reply:
Should be fine now. Was struggling with my IDE to not turn tabs into spaces 😅.